test/extensions/filters/http/jwt_authn/filter_integration_test.cc

```diff
@@ -615,7 +615,7 @@ TEST_P(RemoteJwksIntegrationTest, WithGoodTokenAsyncFetchFast) {
   // the first request will trigger a second jwks fetch, this is not expected, test will fail.
   // To avoid such race condition, before making the first request, wait for the first
   // fetch stats to be updated.
-  test_server_->waitForCounterGe("http.config_test.jwt_authn.jwks_fetch_success", 1);
+  test_server_->waitForCounterGe("http.config_test.jwt_authn.fooprefix.jwks_fetch_success", 1);
```

Comment:
can we define `fooprefix` as a string const so that people can track this easily, as it is shared between two different files (filter_integration_test.cc and test_common.h)

Reply:
this is a good point. but i think a piecemeal parametrization is probably going to make things more confusing since it's less consistent - there are tons of other parts of the config that are shared. I think if i get in there and start to parametrize things, I'll need to end up yak shaving the whole thing to make it clear.

Reply:
I agree that there are lots of other parts doesn't follow the best practice. I feel it is probably fine if they are in the same file , but here is a bit annoying as they are in different file. People who read the code probably will be wondering where does this `fooprefix` come from, which ends up from the test_common.h file/

How about only changing `fooprefix` string that is introduced in this PR? It should be a small change to improve the code readability? WDYT I don't feel very strong about this though